pypureclient/pure1/client.py

```diff
@@ -1,7 +1,11 @@
+from . import Pure1_1_2
+from . import __pycache__
```

Comment:
This __pycache__ import that snuck in seems like it's going to cause problems.

Reply:
Wow, good eye!

Reply:
OK, so this still works (we tested it), but since it's already in pypi, we'll merge this first, and make a 1.42.1 later this week with this cosmetic fix.

Reply:
Current pypi is broken 😬 . 
The changes here look good now, but I think this will have to become 1.42.1 rather than 1.42.0